src/main/java/com/TableTalk/Enterprise/dto/GameControls.java

```diff
@@ -0,0 +1,10 @@
+package com.tabletalk.enterprise.dto;
+
+import lombok.Data;
+
+public @Data class GameControls {
+    private int minPlayers;
+    private int maxPlayers;
+    private int mingameTime;
+    private int maxgameTime;
+}
```

Comment:
Is this class used anywhere?  Are there references to the previous attributes of Game that now refer to GameControls?  Be sure those references are updated.

Also, is there a 1:M relationship between Game and GameControl?  Or, are these attributes of class Game?  If they are all attributes, it probably makes more sense to keep them together, in one class.